examples/dato/README.md

```diff
@@ -0,0 +1,59 @@
+# Makeswift Dato Integration
+
+(Link to landing page):
+https://makeswift-examples-basic-typescript.vercel.app/
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
+## Using this example
+
+To quickly try this example either [deploy to Vercel](#deploy-this-example-to-vercel) or [use our CLI](#use-this-example-locally-with-the-makeswift-cli).
+
+In addition to your Makeswift API key, add your Dato API key in either `.env.local` or your environment variables in vercel.
+
+`NEXT_PUBLIC_DATO_CMS_API_TOKEN=your_token`
+
+---
+
+### Deploy this example to Vercel
+
+The deploy link below includes integrations with Dato and Makeswift.
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmakeswift%2Fmakeswift%2Ftree%2Fmain%2Fexamples%2Fdato&project-name=dato-makeswift-example&repository-name=dato-makeswift-example&redirect-url=https%3A%2F%2Fapp.makeswift.com&integration-ids=oac_51ryd7Pob5ZsyTFzNzVvpsGq,oac_nsrwzogJLEFglVwt2060kB0y&external-id=dato-makeswift)
+
+### Use this example locally with the Makeswift CLI
+
+Set up this example of Makeswift by running
+
+```bash
+npx makeswift@latest init --example=dato
+```
+
+## Local Development
+
+Run `pnpm codegen-ts` after copying the following Dato schema
+
+![alt text](https://github.com/user-attachments/assets/8f235b4b-ed0f-4b99-a018-15ceaa56f1ac)
+
+Note that the Structured Text type ("body") has a validation field under "Specify the blocks allowed" and "Image" in the combobox.
+
+We recommend following the same schema because the `.graphql` files provided assume the same schema.
+
+In order to start visually editing these components in your dynamic page, use the builder's URL bar to navigate directly to the route (i.e. `/blog/my-blog`). This page renders an embedded MakeswiftComponent that represents the Blog Page. You will then be able to use the following component that dynamically renders off the slug:
```

Comment:
This feels like it assumes some Makeswift knowledge specifically of `MakeswiftComponent`. I think we should show an abbreviated code snippet and explain a bit what we've actually done there.

The idea of "dynamically renders off the slug" is pretty high level. I think this is goig to be a new concept for people that view this and we should hand hold them a bit in explaining with this is. Maybe could say, "If you have three blog posts with titles x, y, and z, this will automatically the three associated pages with the following routes".